### PR TITLE
fix( #2034 ): Used Regex in replaceAll methods

### PR DESCRIPTION
**Pull request by yasarshaikh**
### Describe your changes :
Used Regex in replaceAll methods. It was part of #2034. 

This issue was captured in Sonar as a code smell. Fixing it would help to make code good. 

#
### Type of change :

- [x] Bug fix 

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [ ] I have tagged my reviewers below. 
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
 @sureshms

### CODE REVIEW

**Comment by yasarshaikh** [comment]
It seems something wrong with the job instance which is not able to download related dependencies/software to execute all actions.

**Comment by sureshms** [comment]
> It seems something wrong with the job instance which is not able to download related dependencies/software to execute all actions.

There is some intermittent failure unrelated to this.

File: catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/charts/ChartResource.java [hunk 1]
```diff
@@ -111,7 +111,7 @@ public ChartList(List<Chart> data, String beforeCursor, String afterCursor, int
   }
 
   static final String FIELDS = "owner,followers,tags";
-  public static final List<String> FIELD_LIST = Arrays.asList(FIELDS.replaceAll(" ", "").split(","));
+  public static final List<String> FIELD_LIST = Arrays.asList(FIELDS.replaceAll("\\s", "").split(","));
```
Comment (sureshms) [inline catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/charts/ChartResource.java:115]: In this case just replace is sufficient per Sonar Cloud guidelines.

**Review by sureshms — APPROVED** [review]

**Comment by sureshms** [comment]
Since most of these `replaceAll` calls are one time calls, better performance of `replace` call is not important. Merging the change. Thank you @yasarshaikh for your first contribution 🥳 !!!

File: catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/charts/ChartResource.java (hunk 1 above)
Comment (yasarshaikh) [reply catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/charts/ChartResource.java:115]: Let me know if I should change the replaceAll to replace, I can do that it'd be a minor change :)

Comment (sureshms) [reply catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/charts/ChartResource.java:115]: That works as well. You can also create a separate issue to do that.

**Pull request merged**